manifests/init.pp

```diff
@@ -6,23 +6,25 @@
 # Richard Hillmann <richie@project0.de>
 #
```

Comment:
can you please add puppet-strings style documentation for the new parameters?

Reply:
done. I also added a few more to get things going with this module and also generated a REFERENCE.md but it isn't really pretty yet.